addons/ratelmarker/stringtable.xml

```diff
@@ -25,5 +25,23 @@
             <German>RATEL Markierung erstellt (Karte öffnen um zu zentrieren)</German>
             <French>Balise RATEL crée (Ouvrir la carte pour focaliser)</French>
         </Key>
+        <Key ID="STR_TAC_RatelMarker_QRM_settingsCategory">
+            <English>TAC Quick Reference Marker</English>
```

Comment:
Make a new "Markers" category instead and keep them together. Maybe split with sub-categories.